docs/API.md

```diff
@@ -391,16 +397,22 @@ Example request:
 }
 ```
 
-Example response:
+Example response (as generated using the default configuration):
```

Comment:
Have you considered the fact that this will imply that we need to update this part when we add a new language support ?

Reply:
@matsduf Would you object if I remove this?

Reply:
Not at all.